app/assets/src/components/views/SampleUploadFlow/UploadMetadataStep.jsx

```diff
@@ -78,6 +78,15 @@ class UploadMetadataStep extends React.Component {
         <div className={cx(this.state.showInstructions && cs.hide)}>
           <div>
             <div className={cs.title}>Upload Metadata</div>
+            <div className={cs.dictionary}>
```

Comment:
Could you add the link to the Metadata Upload Modal as well? (I think it's <UploadPage>)

Reply:
👍 
![screen shot 2019-03-05 at 3 59 37 pm](https://user-images.githubusercontent.com/5652739/53845902-b2758500-3f5f-11e9-8945-799890ea73e6.png)